### PR TITLE
Allow negative absolute counter values

### PR DESCRIPTION
**Pull request by pirgeo**

### CODE REVIEW

File: lib/src/main/java/com/dynatrace/metric/util/MetricValues.java [hunk 1]
```diff
@@ -34,10 +34,7 @@ private static void throwIfNaNOrInfDouble(double d) throws MetricException {
     private final long value;
     private final boolean isDelta;
 
-    LongCounterValue(long value, boolean isDelta) throws MetricException {
```
Comment (arminru) [inline lib/src/main/java/com/dynatrace/metric/util/MetricValues.java:37]: The Javadoc for `Metric.setLongCounterValueTotal` and `.setDoubleCounterValueTotal` needs to be adapted accordingly.

Comment (arminru) [inline lib/src/main/java/com/dynatrace/metric/util/MetricValues.java:37]: Also the package version needs to be bumped.

File: lib/src/main/java/com/dynatrace/metric/util/Metric.java [hunk 2]
```diff
@@ -76,8 +76,7 @@ public Builder setPrefix(String prefix) {
      *
      * @param value the value to be serialized. Only positive counter values are accepted.
      * @return this
-     * @throws MetricException if the a value has already been set on the metric or if the value is
-     *     less than 0.
+     * @throws MetricException if the a value has already been set on the metric.
```
Comment (dyladan) [inline lib/src/main/java/com/dynatrace/metric/util/Metric.java:79]:
```suggestion
     * @throws MetricException if a value has already been set on the metric.
```

File: lib/src/main/java/com/dynatrace/metric/util/Metric.java [hunk 3]
```diff
@@ -138,8 +137,7 @@ public Builder setLongSummaryValue(long min, long max, long sum, long count)
...
```
Comment (dyladan) [inline lib/src/main/java/com/dynatrace/metric/util/Metric.java:140]:
```suggestion
     * @throws MetricException if a value has already been set on the metric.
```

**Review by dyladan — APPROVED** [review]

**Review by arminru — APPROVED** [review]

**Pull request merged**